### ISSUE TITLE
Add support for abp:// protocol

### ISSUE THREAD

**Issue by leodoi3** [brave/brave-browser#18441]
Please add support for clicking on adb:// links when adding custom filter lists.

**Comment by rebron** [comment]
cc: @antonok-edm

**Comment by antonok-edm** [comment]
Yes, it's on my todo list! I thought we had an issue open for it already, but maybe not - we'll use this one then.
